ext/drmgr/drmgr.h

```diff
@@ -452,6 +465,53 @@ drmgr_unregister_bb_instrumentation_ex_event(drmgr_app2app_ex_cb_t app2app_func,
                                              drmgr_insertion_cb_t insertion_func,
                                              drmgr_ilist_ex_cb_t instru2instru_func);
 
+DR_EXPORT
+/**
+ * Registers callback functions for the third
+ * instrumentation stage: instrumentation
+ * insertion.  drmgr will call \p func for each instruction with the
+ * specific opcode \p opcode.
+ *
+ * More than one call-back function can be mapped to the same opcode. Their
+ * execution sequence is determined by their priority \p priority if set. Ordering
+ * based on priority is also taken in to account with respect to insert bb events.
```

Comment:
suggest: s/insert bb/bb instrumentation/.  Do you mean the main single-instruction instru-insertion event?  If so I would replace "bb" with instruction or sthg.

Reply:
Done.